sources/packages/backend/libs/integrations/src/services/sfas/sims-to-sfas.service.ts

```diff
@@ -119,6 +127,128 @@ export class SIMSToSFASService {
     return modifiedStudentIds;
   }
 
+  /**
+   * Get all student ids and application data of students who have one or more updates in application related data.
+   * @param modifiedSince the date after which the application data was updated.
+   * @param modifiedUntil the date until which the application data was updated.
+   */
+  async getAllApplicationsWithUpdates(
+    modifiedSince: Date,
+    modifiedUntil: Date,
+  ): Promise<ApplicationRecord[]> {
+    return (
+      this.applicationRepo
+        .createQueryBuilder("application")
+        .select("application.id", "applicationId")
+        .addSelect("student.id", "studentId")
+        .addSelect("programYear.programYear", "programYear")
+        .addSelect("offering.offeringIntensity", "offeringIntensity")
+        // Use CASE to conditionally select studyStartDate and studyEndDate and casting JSON values to dates.
+        .addSelect(
+          `CASE
+      WHEN studentAssessment.offering IS NULL THEN (application.data->>'studyStartDate')::date
```

Comment:
The property from formio has start with s in lowercase. 

![image](https://github.com/user-attachments/assets/3edab843-0548-4a62-9c53-c1271573ebba)